go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/snowplow/snowplow-golang-tracker/v2
```

Comment:
I think we need to change this to v3 (the PR is for release 3.0.0). I guess this also means updating the module reference in all the source files that use any of the pkg packages. Sounds a bit annoying but not sure if there is a better way to not have to repeat the version in all imports moving forward.

Reply:
I just realized that this will probably be a part of a "Prepare for release" commit, didn't notice that there wasn't a commit like that yet.

Reply:
Done now!